Marketplace/voiceHUDApp/appResources/appData/resources/voiceHUD_ui.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="https://fonts.googleapis.com/css?family=Raleway:300,400,600,700" rel="stylesheet">
+        <link rel="stylesheet" type="text/css" href="./css/styles.css">
+    </head>
+
+    <body>
+        <div id="info">
+            <h1>Voice HUD</h1>            
+        </div>
+        
+        <div id="instructions">
+            <p> Click the button below to see others' voices as a floating overlay, click again to turn it off.  <br><br>Users who are quiet for 5 seconds or more will not show anything</p>
+        </div>
+
+        <div id="main-area">
+            <input type="button" id="toggle" value="">
```

Comment:
Using buttons to show state is usually a confusing UX paradigm. Please make this a switch, or separate out "on" and "off" buttons. We can talk more about this in-person if you'd like.